build/packaging/rpm/glideinwms.spec

```diff
@@ -1109,6 +1111,11 @@ rm -rf $RPM_BUILD_ROOT
 
 
 %changelog
+* Sat Mar 22 2025 Marco Mambelli <marcom@fnal.gov> - 3.10.11
```

Comment:
Should the date change to `Mar 24 2025`?

Reply:
RC2 was on Saturday. I'll change for the final release once I get to that